pages/home.html

```diff
@@ -552,4 +555,4 @@ <h2 class="text-center">Sponsored by</h2>
 	});
 </script>
 
-{% include footer.html %}
+{% include footer.html %}
```

Comment:
Needs a new line at the end of the file, which Akien added in his recent PR, and this PR removes again.

Reply:
Missed that one! I did not update my local `master` when creating the branch. Should be good now 👍